src/xdocs/google_style.xml

```diff
@@ -2435,6 +2435,30 @@
           </table>
         </div>
       </subsection>
+      <subsection name="Overrides" id="Google_Overrides">
+        <p>
+          To change the default severity level, use the following custom property variable:
+          <br/>
+          <br/>
+          <code>org.checkstyle.google.severity=${SeverityLevel}</code>
+        </p>
+        <p>
+          Visit here for more details about Severity Levels:
+          <a href="https://checkstyle.org/property_types.html#SeverityLevel">
```

Comment:
Covered by other comment but do not use absolute links for the web pages if it links to our own documentation. It breaks version documentation.

Reply:
done. removed absolute links